lib/php/lib/Protocol/TJSONProtocol.php

```diff
@@ -297,21 +233,14 @@ private function writeJSONDouble($num)
             $this->trans_->write(self::QUOTE);
         }
 
+        #TODO add compatibility with for NAN and INF
```

Comment:
if we will try to make json_encode on such values we will get a silent error, whic can be readed only by method json_get_last_error. In future should be added flag THROW_ERROR_ON_EXCEPTION

Reply:
https://issues.apache.org/jira/browse/THRIFT-5770